### ISSUE TITLE
Properties registered on the site do not show on search places

### ISSUE THREAD

**Issue by LunaAsefaw** [kartoza/sawps#1177]
This organisation has two properties called 'test' so when you search for properties that exist, they exist in the drop-down (since I can see two different lines highlighted when I hover over them) but they do not show in the dropdown. This is seen for all registered properties

https://github.com/kartoza/sawps/assets/90245715/325e8459-3939-4b33-ac09-0f9edd1715b5

**Comment by danangmassandy** [comment]
@LunaAsefaw this should be fixed in PR https://github.com/kartoza/sawps/pull/1462.
However, I have question, when we search for a property, we can only see property that belongs to our organisations right? regardless the active organisation or not.
I think #1367 is also related to current active organisation, the current map implementation in staging which is without filter, it will only draw properties from current active organisation.

**Comment by LunaAsefaw** [comment]
@danangmassandy Yes, I should see all the properties whose organisation I am part of, regardless on if it is active or now.
 The property mentioned in #1367 is Hluhluwe-Imfolozi Park and that belongs to Cape Nature. When I did the test, I set Cape nature to active so I am not sure if that issue is due to another organisation being active.
Also with this ticket, it actually does list properties on the site but it hides then, however I can type in something and click on the empty lists and it will send me to the property with a name like the one I inputted
